lib/cogctl/action_util.ex

```diff
@@ -47,12 +73,20 @@ defmodule Cogctl.ActionUtil do
       {:ok, endpoint_with_token} ->
         fun.(endpoint_with_token)
       {:error, error} ->
+        error = case error do
+                  error when is_list(error) ->
+                    Enum.join(error, "\n")
+                  error ->
+                    inspect(error)
+                end
+
```

Comment:
using cond may be clearer here to see what you are doing and you aren't using error as input and output.
```elixir
{:error, errors} ->
  error = cond do
    is_list(errors) -> Enum.join(errors, "\n")
    true -> inspect(errors)
  end
```